src/gcp/cloudfunctions.ts

```diff
@@ -307,7 +335,7 @@ export async function setInvoker(
   const policy: iam.Policy = {
     bindings: bindings,
     etag: currentPolicy.etag || "",
-    version: currentPolicy.version || 3,
+    version: 3,
   };
   await setIamPolicy({ name: fnName, policy: policy });
```

Comment:
One more thing I just realized: it might be nice to not update the bindings if they're already set correctly (the 99% case). This will not only save API requests, but avoid triggering scary sounding audit logs.